app/controllers/registrations_controller.rb

```diff
@@ -31,7 +31,11 @@ def create
       yield resource if block_given?
       if resource.persisted?
         update_first_user_permissions(resource)
-        redirect_to "/confirm-email?email=#{CGI.escape(resource.email)}"
+        if SiteConfig.smtp_enabled?
+          redirect_to "/confirm-email?email=#{CGI.escape(resource.email)}"
```

Comment:
should we use a path helper here? 

```ruby
> confirm_email_path(email: "bob@acme.com")
=> "/confirm-email?email=bob%40acme.com"
```

Rails will take care of escaping it ;-)

Reply:
Great idea. Thank you!
```suggestion
          redirect_to confirm_email_path(email: resource.email)
```